### PR TITLE
Add .NET 6.0 to requirements and Add nuke global tool as optional tool.

### PR DESCRIPTION
**Pull request by Kielek**
## Why

Keep dev documentation up to date.

## What

Add .NET 6.0 to requirements.
Add nuke global tool as optional tool.

## Tests

N/A

### CODE REVIEW

File: docs/DEVELOPING.md [hunk 1]
```diff
@@ -30,6 +32,13 @@ To see a list of possible targets run:
 .\build.cmd --help
```
Comment (pellared) [inline docs/DEVELOPING.md:32]: these commands are incorrect. it should be `.\tracer\build.cmd` 😉

**Review by pellared — APPROVED** [review]

File: docs/DEVELOPING.md [hunk 2]
```diff
@@ -12,15 +12,17 @@ Minimum requirements:
...
   - Individual components
     - .NET Framework 4.7 targeting pack
-- [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
-- [.NET 5.0 x86 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) to run 32-bit tests locally
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [.NET 6.0 x86 SDK](https://dotnet.microsoft.com/download/dotnet/6.0) to run 32-bit tests locally
 - Optional: [ASP.NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET Core 2.1 locally.
 - Optional: [ASP.NET Core 3.0 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.0) to test in .NET Core 3.0 locally.
 - Optional: [ASP.NET Core 3.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.1) to test in .NET Core 3.1 locally.
+- Optional: [ASP.NET 5.0 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET 5.0 locally.
```
Comment (lachmatt) [inline docs/DEVELOPING.md:20]: link points to the wrong version

File: docs/DEVELOPING.md (hunk 1 above)
Comment (Kielek) [reply docs/DEVELOPING.md]: fixed.

File: docs/DEVELOPING.md (hunk 2 above)
Comment (Kielek) [reply docs/DEVELOPING.md]: fixed

**Pull request merged**